articles/aggregation-in-dbms/index.md

```diff
@@ -0,0 +1,78 @@
+Database management systems (DBMS) have replaced the traditional filing system by providing an easy, secure, efficient, and reliable way of storing, retrieving, accessing, and sharing data within databases. Some entities in a DBMS may have little value, which makes it difficult to use them for certain operations. In such situations, these entities can be combined with other entities to form a complex one that makes sense. This operation can be done through a process called aggregation. Aggregation in DBMS links trivial entities through relationships to ensure that the entire system functions well. 
+
+This article takes you through important aspects regarding aggregation in DBMS. It also explains why it is needed and how it takes place. 
+

```

Comment:
It would be useful if you could add a Table of Contents to this article. For example, check out

https://raw.githubusercontent.com/section-io/engineering-education/master/articles/binary-tree-data-structure-python/index.md

Reply:
Done.